README.md

```diff
@@ -204,7 +207,79 @@ Creating a voice requires an achievable, but significant effort. An individual w
+
+## Query database structure
+Mimic-Recording-Studio writes all recordings in a sqlite database file located under /backend/db/. This can be opened with database tools like DBeaver.
+
+The database includes two tables.
+
+![database_table_overview](./img/database_table_overview.png)
+
+### Table "audiomodel"
+All recordings are persisted in this table with 
+* recording timestamp (created_date)
+* uuid of speaker (matches the filesystem path under /backend/audio_files/id)
+* wav filename in filesystem (audio_id)
+* text of recorded phrase (phrase)
+
+The database can be used to query your recordings.
+
+Here are some example queries:
+
+```sql
+-- List all recordings
+SELECT * FROM audiomodel;
+
+-- Lists recordings from january 2020 order by phrase
+SELECT * FROM audiomodel WHERE created_date BETWEEN '2020-01-01' AND '2020-01-31' ORDER BY prompt;
+
+-- Lists number of recordings per day
+SELECT DATE(created_date), COUNT(*) AS RecordingsPerDay
+FROM audiomodel
+GROUP BY DATE(created_date )
+ORDER BY DATE(created_date)
+
+-- Shows average text length of recordings
+SELECT AVG(LENGTH(prompt)) AS avgLength FROM audiomodel
+```
+
+There are many ways more on how query sqlite database might help you but these should be enough to show the possibilities.
+> E.g. Looking for recordings in a specific timerange might help to remove recordings made in a bad environment.
+
+### Table "usermodel"
+Even if Mimic-Recording-Studio just uses one sqlite database it can be used by more than one speaker.
+
+This tables provides following informations per speaker:
+* Unique identifier of speaker (uuid)
+* Name of speaker (user_name)
+* Newest recorded line number of corpus (propmpt_num)
+* Total recording time (total_time_spoken)
+* How many chars have been recorded (len_char_spoken)
+
+These values are e.g. used to calculate the speaking pace to show if the recorded phrase is too fast or slow compared to previous recordings.
+
+Query table "usermodel" to get a list of speakers and some recording statistics on them.
+
+## Modify recorder uuid in browser
+The browser used to record your phrases persists the user uuid and user_name in it's local storage.
+
+> Values in browsers local storage, file system structure and sqlite database should always match!
+
+If a problem occurs and your browser looses/changes uuid mapping for Mimic-Recording-Studio you could have difficulties to continue a previous recording session.
+
+If this happens it seems easiest to update local storage of your browser. For that you need the following two informations:
+
+* uuid (query "usermodel" in sqlite or check filesystem path under /backend/audio_files/)
+* user_name (query "usermodel")
+
+Open Mimic-Recording-Studio in your browser, jump to web-developer options, local Storage and set name and uuid to the original values.
+
+![browser_local_storage](./img/browser_local_storage.png)
+
+After that you should be able to continue your previous recording session without further problems.
+
```

Comment:
## Modify recorder uuid
The browser used to record your phrases stores a unique identifier (`uuid`) and name (`user_name`) in it's localStorage.

If a problem occurs and your browser loses that localStorage then Mimic-Recording-Studio will ask for a new user name and generate a new uuid. This "new" user not match your previous recordings and you will be prompted to start from the beginning.

Instead of updating the database, it is easier to revert the values in the localStorage of your browser. The aim is to match the previous values that are still present in your sqlite database and the file system structure of your existing recordings. To find the previous values for:
* `uuid` - query "usermodel" table in sqlite, or check the filesystem path under `/backend/audio_files/`
* `user_name` - query "usermodel" table in sqlite

Once you have the correct values, open Mimic-Recording-Studio in your browser, then open the browser Web Developer Tools and find localStorage.
* In Firefox this is located in the Storage tab.
* In Chrome this is located in the Application tab.

![browser_local_storage](./img/browser_local_storage.png)

Now set your `user_name` and `uuid` to the original values and you should be able to continue your previous recording session.

Reply:
What did you think about this one? 
I think the main things are:
- we can use either "localStorage" or "Local Storage" - both nouns. But not "local Storage" unless you are referencing "a Storage object that is local" eg local is an adjective. This may or may not be the "localStorage" object.
- I'm not keen on ">" quote formatting to highlight a point. It suggests that the content is from somewhere else.
- Try to strengthen the language of "If this happens it seems easiest" eg "it is easier"

All great additions though, especially given English isn't your primary language!

Reply:
Thanks Kris for your constructive feedback.
I've modified to use "localStorage" all over and updated some wordings.